app/src/lib/stores/helpers/ahead-behind-updater.ts

```diff
@@ -100,25 +100,22 @@ export class AheadBehindUpdater {
     this.clear()
 
     const from = currentBranch.tip.sha
+    const cache = this.comparisonCache
+    const newRefs = new Set<string>()
 
-    const filterBranchesNotInCache = (branches: ReadonlyArray<Branch>) => {
-      return branches
-        .map(b => b.tip.sha)
-        .filter(to => !this.comparisonCache.has(from, to))
+    const addBranch = (b: Branch | null) => {
+      if (b && b.isDesktopForkRemoteBranch && !cache.has(from, b.tip.sha)) {
```

Comment:
If I'm following this PR's logic correctly, shouldn't this filter *out* Desktop fork remote branches? 🤔 
```suggestion
      if (b && !b.isDesktopForkRemoteBranch && !cache.has(from, b.tip.sha)) {
```

Reply:
One day I'll learn how to computer. Great catch @sergiou87!